src/jdk.jfr/share/classes/jdk/jfr/internal/EventControl.java.rej

```diff
@@ -0,0 +1,29 @@
+diff a/src/jdk.jfr/share/classes/jdk/jfr/internal/EventControl.java b/src/jdk.jfr/share/classes/jdk/jfr/internal/EventControl.java	(rejected hunks)
```

Comment:
This checks in `.rej` file! Odd that bots have not complained about it. Please report this to Skara folks.